.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: java
+sudo: false
+jdk:
+  - oraclejdk7
+install:
+  - wget http://www.eclipse.org/downloads/download.php?file=/eclipse/downloads/drops/R-3.6.2-201102101200/eclipse-SDK-3.6.2-linux-gtk-x86_64.tar.gz -O eclipse-3.6.2-linux64.tar.gz
+  - tar xzvf eclipse-3.6.2-linux64.tar.gz eclipse
+  - echo eclipsePlugin.dir=$(pwd)/eclipse/plugins > eclipsePlugin/local.properties
+script: ant build test
+notifications:
+  email:
+    recipients:
+      - skypencil+travis@gmail.com
```

Comment:
In my [personal project](https://github.com/amaembo/streamex/blob/b170723dec9ae866f64c78e1b83f54e981ffba55/.travis.yml) I have no mails configured, but they are nevertheless sent automatically to me when build is failed or new branch is built. I guess, project members will receive the mails automatically here as well.